static/js/components/NewsFeed.jsx

```diff
@@ -4,7 +4,7 @@ import { Link } from "react-router-dom";
 import { useSelector } from "react-redux";
 import PropTypes from "prop-types";
 
-import { Paper, Avatar, Tooltip } from "@mui/material";
+import { Avatar, Paper, Tooltip } from "@mui/material";
```

Comment:
Ah can you actually change that one import of multiple elements to 3 distinct direct imports like we use for everything else? These can increase the webpack size, and looks like we missed that one!

Reply:
✅ Done.